app/controllers/users_controller.rb

```diff
@@ -6,6 +6,7 @@ def new
   def create
     @user = User.new user_params
     if @user.save
+      UserMailer.with(user: self).welcome_email.deliver_now
```

Comment:
Theo em thì `self` ở đây có giá trị là gì?

Reply:
Em hãy test ở máy của em xem khi gửi mail thì nội dung in ra có đúng như em muốn không nhé

Reply:
da e test r va no k chay dc thay bang ```@user``` thi chay dc a